src/test/storage/table_test.cpp

```diff
@@ -1,5 +1,5 @@
+#include <thread>
 #include "base_test.hpp"
-
```

Comment:
Not too sure about these changes - are they needed?

Reply:
See above